requirements/base.in

```diff
@@ -10,6 +10,7 @@ django==1.11.26
 django-compressor==2.2
 django-cors-headers                 #  Used to allow to configure CORS headers for cross-domain requests
 django-crispy-forms
+django-crum
```

Comment:
Would it be useful to pin django-crum version?? @andrey-canon

Reply:
the version is specified in the txt file, the .in file shouldn't contain pinned versions